include/swift/AST/Decl.h

```diff
@@ -958,6 +958,10 @@ class alignas(1 << DeclAlignInBits) Decl {
     assert(Mem); 
     return Mem; 
   }
+
+  /// Scaffolding to permit experimentation with finer-grained dependencies and
+  /// faster rebuilds.
+  bool getEnableExperimentalDependencies() const;
```

Comment:
Are you expecting this to be a per-Decl or per-DeclContext setting? If not, please don't do this; just stick to the longer `getASTContext().LangOpts. EnableExperimentalDependencies`.

Reply:
OK, gone.